HostApplication/AppDelegate.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AppDelegate.h
+//  HostApplication
+//
+//  Created by Hrishikesh Amravatkar on 8/8/19.
+//  Copyright © 2019 Leanplum. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface AppDelegate : UIResponder <UIApplicationDelegate>
```

Comment:
this PR is to add caching models. It's also adding a `HostApplication` . We should discuss what the purpose of that is...

Reply:
Sure. The reason for HostApplication is for Keychain testing to work. The unit test cases for Keychain need to have HostApplication as Keychain is application context.

Reply:
sounds good